qtaf-core/src/main/java/de/qytera/qtaf/core/gson/GsonFactory.java

```diff
@@ -21,27 +29,111 @@ private GsonFactory() {
      */
     private static Gson instance = null;
 
+    private static Gson instanceWithoutCustomSerializers = null;
+
+    /**
+     * JsonSerializer instances are saved in this map
+     */
+    private static final Map<String, IQtafJsonSerializer> serializers = new HashMap<>();
+
+    /**
+     * ExclusionStrategy instances are saved in this map
+     */
+    private static final List<ExclusionStrategy> exclusionStrategies = new ArrayList<>();
```

Comment:
I think this could cause errors because the configuration class needs to be instantiated first, otherwiese GsonFactory will not know in which packages to look for classes